src/App/authActions.ts

```diff
@@ -22,6 +24,20 @@ async function authFunc(body: GoogleBody, props: AppProps): Promise<string | Err
     data = await superagent.post(`${process.env.BackendUrl}/user/auth/google`)
       .set({ Accept: 'application/json' }).send(body);
   } catch (e) {
+    props.dispatch(authError((e as Error)));
+    store.addNotification({
+      title: (e as Error).message,
+      message: 'Error, cannot dispatch',
+      type: 'warning',
+      insert: 'top',
+      container: 'top-right',
+      animationIn: ['animate__animated animate__fadeIn'],
+      animationOut: ['animate__animated animate__fadeOut'],
+      dismiss: {
+        duration: 5000,
+        onScreen: true,
+      },
+    });
     props.dispatch(authError(e as Error));
     return Promise.reject(e);
   }
```

Comment:
on line 46 before that line you need another error dialog there for 'Authentication Failed'